Jenkins/update_ci_win_docker_image.ps1

```diff
@@ -0,0 +1,83 @@
+
+$ErrorActionPreference = "Stop"
+
+$globalVariables = (Resolve-Path "$PSScriptRoot\..\global-variables.ps1").Path
+$ciUtils = (Resolve-Path "$PSScriptRoot\..\Modules\CIUtils").Path
+
+Import-Module $ciUtils
+. $globalVariables
+
+$WORKSPACE="$env:WORKSPACE"
+$DOCKER_HUB_USER="$env:DOCKER_HUB_USER"
+$DOCKER_HUB_USER_PASSWORD="$env:DOCKER_HUB_USER_PASSWORD"
+$DOCKERFILE_PATH="$PSScriptRoot\ci-image-Dockerfile"
+$IMAGE_FILES_DIR="$WORKSPACE\files"
+$IMAGE_FILES_BASE_URL="$LOG_SERVER_BASE_URL/downloads"
+
+if (!$DOCKER_HUB_USER) {
+    Write-Output "Docker user is not set."
+    exit 1
+}
+
+if (!$DOCKER_HUB_USER_PASSWORD) {
+    Write-Output "Docker user password is not set."
+    exit 1
+}
+
+# Do a login into docker
+Write-Output $DOCKER_HUB_USER_PASSWORD | docker login -u $DOCKER_HUB_USER --password-stdin
+
+pushd $WORKSPACE
```

Comment:
We can remove this as we do everything in the `$IMAGES_FILES_DIR`. There is a `pushd` for this below.